src/components/Quiz/QuizWidget/index.tsx

```diff
@@ -0,0 +1,157 @@
+
+  const { quizKey, isStandaloneQuiz = false } = props
+
+  const getMainContainerBg = (): StackProps["bg"] => {
+    if (!showAnswer) {
+      return "neutral"
+    }
+
+    if (currentQuestionAnswerChoice?.isCorrect) {
+      return "success.neutral"
+    }
+
+    return "error.neutral"
+  }
+
+  // Render QuizWidget component
+  return (
+    <VStack spacing="12" width="full" maxW="600px">
+      {/* Hide heading if quiz is not in Learning Quizzes Hub page */}
+      {isStandaloneQuiz && <StandaloneQuizHeading />}
+      {/* Inner Container */}
+      <Stack
+        w="full"
+        gap="8"
+        px={{ base: "8", md: "12", lg: "16" }}
+        // Reduce padding when showing Spinner
+        pt={!quizData ? "10" : { base: "5", md: "12" }}
+        pb={!quizData ? "5" : { base: "4", md: "8" }}
+        bg={getMainContainerBg()}
+        borderRadius="base"
+        boxShadow={isStandaloneQuiz ? "drop" : "none"}
+        position="relative"
+        isolation="isolate"
+      >
+        {showConfetti && <QuizConfetti />}
+
+        <Center
+          position={{ base: "relative", md: "absolute" }}
+          top={{ base: 2, md: 0 }}
+          left={{ md: "50%" }}
+          transform="auto"
+          translateX={{ md: "-50%" }}
+          translateY={{ md: "-50%" }}
+        >
+          <AnswerIcon
+            showAnswer={showAnswer}
+            isCurrentQuestionCorrect={currentQuestionAnswerChoice?.isCorrect}
+          />
+        </Center>
+        <Stack spacing="8" justifyContent="space-between">
+          {quizData ? (
+            <>
+              <QuizContent
+                showAnswer={showAnswer}
+                isCurrentQuestionCorrect={
+                  currentQuestionAnswerChoice?.isCorrect
+                }
+                currentQuestionIndex={currentQuestionIndex}
+                title={quizData.title}
+                questions={quizData.questions}
+                showResults={showResults}
+                quizSummaryProps={{
+                  quizKey: quizKey!,
+                  numberOfCorrectAnswers,
+                  isPassingScore,
+                  questionCount: quizData.questions.length,
+                  ratioCorrect,
+                  quizScore,
+                  setUserStats,
+                }}
+                quizRadioGroupProps={{
+                  handleSelection,
+                  selectedAnswer,
+                }}
+                progressBarBackground={progressBarBackground}
+              />
+              <QuizButtonGroup
+                currentQuestionAnswerChoice={currentQuestionAnswerChoice}
+                finishedQuiz={finishedQuiz}
+                handleContinue={handleContinue}
+                handleNextQuiz={handleNextQuiz}
+                handleRetryQuestion={handleRetryQuestion}
+                handleSubmitAnswer={handleSubmitAnswer}
+                handleShare={handleShare}
+                hasNextQuiz={hasNextQuiz}
+                initialize={initialize}
```

Comment:
I'd call this `reset` since this is called when the user clicks on since "Try again" button.

Reply:
This was an existing prop, but I can look at updating this 😁

Reply:
Changed it to `handleReset` since it is a handler sent to `onClick`